src/actions/constants.js

```diff
@@ -1,3 +1,7 @@
+import { LOCATION_CHANGE } from 'react-router-redux'
+
+export { LOCATION_CHANGE }
```

Comment:
in the other file where we reference location change, we just import it directly in that file from react-router-redux